Makefile

```diff
@@ -90,11 +90,14 @@ test: ## Run all unit tests
 	@echo "Unit tests:"
 	@go test -race -covermode=atomic -coverprofile=coverage-unit.out -timeout 1s -tags=unit ./...
 	@echo "Integration tests:"
-	@set -o pipefail && go test -race -covermode=atomic -coverprofile=coverage-integration.out -timeout 15s -tags=integration ./... | grep -v '\[no test files\]'
+	@set -o pipefail && go test -race -covermode=atomic -coverprofile=coverage-integration.out -timeout 15s -tags=integration ./...
 # Given the nature of generative tests the test timeout is increased from 500ms
 # to 30s to accommodate many samples being generated and test cases being run.
 	@echo "Generative tests:"
-	@set -o pipefail && go test -race -covermode=atomic -coverprofile=coverage-generative.out -timeout 30s -tags=generative ./... | grep -v '\[no test files\]'
+	@set -o pipefail && go test -race -covermode=atomic -coverprofile=coverage-generative.out -timeout 30s -tags=generative ./...
+
+quieter-test:
```

Comment:
```suggestion
quieter-test: ## Run tests a bit less verbose
```

Reply:
One idea: We could make this the default for `make test`. I wasn't sure if we want to always hide the 0.0% output, so that's why I didn't do that.

Reply:
Note that the `grep -v '\[no test files\]'` that I removed does almost nothing, there's only one of those lines, this:

`github.com/enterprise-contract/ec-cli/internal/rego`